app/graphql/types/business/mutation.gql

```diff
@@ -0,0 +1,7 @@
+extend type Mutation {
+  updateBusinessDetails(input: UpdateBusinessDetailsInput!): UpdateBusinessDetailsResponse @on
+}
+
+type UpdateBusinessDetailsResponse {
+  success: Boolean @on
```

Comment:
I know that in the the interest of time this can be done later, but to capture something from our Slack conversation…

I'd suggest that it would also be a good idea to return some usable data from a mutation to allow consumers to get updated data without having to re-fetch. See https://www.apollographql.com/blog/designing-graphql-mutations for examples.

```suggestion
  success: Boolean @on
  business: Business @on
```